pkg/logger/logger.go

```diff
@@ -43,6 +46,15 @@ func (l *logger) AddFields(fields ...Field) {
 }
 
 func (l *logger) Sync() {
+	l.Info("syncing logger")
```

Comment:
this was just for debugging right? doesnt look like we still need it

Reply:
Yeah good catch